libtest/cr_template.c

```diff
@@ -39,8 +58,6 @@ void
 deinit_template_tests(void)
 {
   deinit_syslogformat_module();
-  if (!testutils_deinit())
```

Comment:
This was the last user of `testutils_deinit`.
As we try to move on to criterion from libtest: should we delete those funcs one-by-on from testutils or we will remove everything in one step when the last testcase depending on testutils is removed?